python_modules/libraries/dagster-dg/dagster_dg/cli/plus/__init__.py

```diff
@@ -514,3 +374,18 @@ def create_ci_api_token(description: Optional[str] = None, **global_options: obj
         gql.CREATE_AGENT_TOKEN_MUTATION, variables={"description": description}
     )
     click.echo(token_data["createAgentToken"]["token"])
+
+
+@click.group(
```

Comment:
what's the reason for inverting the group pattern here? seems fine, just curious

Reply:
What do you mean by inverting the group pattern?

Reply:
oh, just building a group by specifying its commands rather than calling `@my_group.command` on each one